ui/src/app/views/workspace/component.html

```diff
@@ -24,15 +24,15 @@ <h3 class="table-header-descriptor">Cohorts</h3>
   </div>
   <div class="indented">
     <clr-datagrid [clrDgLoading]="cohortsLoading">
-      <clr-dg-column [clrDgField]="'cohort.name'" [clrDgSortBy]="cohortNameComparator" [style.width.%]="25">
+      <clr-dg-column [clrDgField]="'cohort.name'" [clrDgSortBy]="cohortNameComparator" class="quarter-table">
```

Comment:
Hmm... I actually find `[style.width.%]="25"` to be slightly clearer than `class="quarter-table"` with the added benefit that I don't have to go look something up. Other opinions?

Reply:
My ideal would be relative weights for columns. That way, when you add another column, it rebalances on its own; you don't have to do arithematic. (That's what I was thinking, regarding flexbox above, though I forget if it actually works with tables -- it may not apply.) But if it's fixed values, and they're all in one place anyway, I don't think pulling the fixed value out to a stylesheet saves us anything.

If/when we switch to SCSS/SASS, we should see if the pre-processing also gets us some validation; and if so, whether that applies only to separated stylesheets or if it can evaluate style values in templates too. If the former, that would be a reason to pull more things out. (Actually, adding SCSS would be a great candidate task now.)

Reply:
I can look into doing flexbox for now. I agree that I think SCSS is a good thing to try to include while we have the time.

Reply:
Re: David Mohs, I think I agree that the [style.width.%] is more clear. (And about the same number of characters as assigning the class.